system-tests/test/record_spec.js

```diff
@@ -2354,20 +2354,13 @@ describe('e2e record', () => {
             spec: 'simple_multiple.cy.js',
             configFile: 'cypress-with-project-id.config.js',
             record: true,
-            snapshot: true,
+            snapshot: false,
             expectedExitCode: 1,
           }).then(() => {
             const requests = getRequests()
             const postResultsRequest = requests.find((r) => r.url === `POST /instances/${instanceId}/results`)
 
-            console.log(postResultsRequest)
             expect(postResultsRequest?.body.exception).to.include('Your configFile threw an error')
-            expect(postResultsRequest?.body.tests).to.have.length(2)
```

Comment:
@cacieprins I'd suppose the original intent of this was just to make sure we didn't blow away the results when there's an error and that we exit properly. Seems like you could still keep the `body.tests - to have length 2`..... is that assertion really adding more value than not having it though? 🤔

Reply:
Sometimes the before spec listener can error before the reporter knows how many tests there are!